sources/core/Stride.Core.CompilerServices.Tests/AnalyzerTests/ValidObjectCombinations.cs

```diff
@@ -0,0 +1,87 @@
+using Xunit;
+
+namespace Stride.Core.CompilerServices.Tests.AnalyzerTests;
+
+public class ValidObjectCombinations
```

Comment:
I feel like this could be named better as something like `ValidModifiersOnMutableMemberTests` and then maybe the other file as `ValidModifiersOnImmutableMemberTests`